tests/gnrc_dhcpv6_client/Makefile

```diff
@@ -0,0 +1,53 @@
+include ../Makefile.tests_common
+
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h nucleo32-f042 \
+                             nucleo32-f031 nucleo-f030 nucleo-l053 \
+                             nucleo32-l031 stm32f0discovery telosb \
+                             wsn430-v1_3b wsn430-v1_4 z1
+
+# The following boards do not have an available UART
+BOARD_BLACKLIST += mips-malta pic32-wifire pic32-clicker ruuvitag thingy52
+
+TEST_ON_CI_WHITELIST += all
```

Comment:
the sudo dependency unfortunately makes this impossible. ```TEST_ON_CI_WHITELIST``` is supposed to list all boards on which it makes sense to run this test without any dependencies.

Reply:
Then how do I whitelist this test for Murdock?

Reply:
The test will always be compiled (wrt the BOARD_BLACK/WHITELIST vars). The TEST_ON_CI_WHITELIST variable makes murdock run "make test".

"make test" won't work inside the build containers for this test as it requires sudo and tap and probably more.

Reply:
Yes, I know that `TEST_ON_CI_WHITELIST` is running `make test`. My question was, how can we make such tests possible on Murdock? This isn't such an exotic use case. As soon as you want to test some packet injecting or analysis for network traffic (which is the a sensible way to test networking automated, beyond the dull "let's see if we can send some [valid] packets around") you need to bind to a tap interface, which requires `sudo`. Beyond that, yes I use `scapy` which is a nifty python tool and library to generate and analyse network traffic ;-) (and requires sudo to do that on Ethernet packet, which you need to as soon as TAP interfaces are involved).

Reply:
That's what the improved testrunner will cover.

Reply:
Ok, then I remove this line for now.